### PR TITLE
[AzureStaticWebAppV0]  Added pull so newest version of docker image is always used

### PR DESCRIPTION
**Pull request by tomaustin700**
**Task name**: AzureStaticWebAppV0

**Description**: Added pull so newest version of docker image is always used

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y) #15629 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by nwaf999** [comment]
H

**Review by miwebst — APPROVED** [review]

**Comment by tomaustin700** [comment]
Hi @miwebst, have merged master if you want to review again. Thanks.

**Review by miwebst — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/AzureStaticWebAppV0/task.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "demands": [],
   "version": {
     "Major": "0",
-    "Minor": "194",
-    "Patch": "2"
+    "Minor": "198",
```
Comment (alexander-smolyakov) [inline Tasks/AzureStaticWebAppV0/task.json:17]: Could you please bump the task version up to 200?

**Review by alexander-smolyakov — APPROVED** [review]

File: Tasks/AzureStaticWebAppV0/task.json (hunk 1 above)
Comment (tomaustin700) [reply Tasks/AzureStaticWebAppV0/task.json]: @alexander-smolyakov All done

**Comment by alexander-smolyakov** [comment]
@tomaustin700 thanks! We've merged the PR. The related changes will be available in the 200th sprint release.

**Pull request merged**